tests/task_signal_test.py

```diff
@@ -52,18 +52,15 @@ def setUp(self):
     @mock.patch('girder.utility.model_importer.ModelImporter')
     @mock.patch('girder.api.rest.getCurrentUser')
     def test_girder_before_task_publish_with_jobinfospec_no_job_created(self, gcu, mi):
-
         inputs = dict(sender='test.task',
                       # args, kwargs, options
                       body=[(), {}, {}],
                       headers=self.headers)
 
         create_job = mi.model.return_value.createJob
-
         with mock_worker_plugin_utils():
             girder_before_task_publish(**inputs)
-
-        self.assertTrue(not gcu.called)
+        gcu.assert_called_once()
```

Comment:
Sorry,  can you explain why this needed to change?  Thanks!

Reply:
@kotfic Before the token we use was orphan, now we specify a user to the token. Thus getCurrentUser gets called so that we can attach it to the token.